WalletWasabi.Gui/Controls/WalletExplorer/TransactionInfoTabViewModel.cs

```diff
@@ -1,16 +1,33 @@
+using System;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using ReactiveUI;
+using Splat;
 using WalletWasabi.Gui.Controls.TransactionDetails.ViewModels;
 using WalletWasabi.Gui.ViewModels;
 
 namespace WalletWasabi.Gui.Controls.WalletExplorer
 {
 	public class TransactionInfoTabViewModel : WasabiDocumentTabViewModel
 	{
-		public TransactionInfoTabViewModel(TransactionDetailsViewModel transaction) : base("")
+		public TransactionInfoTabViewModel(TransactionDetailsViewModel transaction, UiConfig uiConfig) : base("")
```

Comment:
It's good to add parameter names in cases like this: `base("")` (meaning: `base(parameterName: "")` because empty string is not self-explaining.

Reply:
Done.